### PR TITLE
Enable source maps in production

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/next.config.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ const nextConfig = {
    */
   reactStrictMode: true,
 
+  /**
+   * Enable source maps in production, because we want people to report readable
+   * stack traces from the error boundaries feature.
+   */
+  productionBrowserSourceMaps: true,
```
Comment (VWSCoronaDashboard10) [inline packages/app/next.config.js:19]: does this out of the box write pretty stack traces to the error dumps?

Comment (VWSCoronaDashboard1) [reply packages/app/next.config.js:19]: AFAIK, if sourcemaps are configured they automatically end up in your stacktrace.

"Source maps are natively supported by both Google Chrome and Firefox, as well as IE11. This means that any minified JavaScript files with a source map comment at the bottom will automatically link to it’s sources, so long as the mapping and source files are in the correct locations."

Comment (VWSCoronaDashboard8) [reply packages/app/next.config.js:19]: Yes that's what I'm hoping for at least :)

Comment (VWSCoronaDashboard10) [reply packages/app/next.config.js:19]: wonderful ✨

**Pull request merged**